src/test/java/stepDefinition/ProductPageStepDef.java

```diff
@@ -0,0 +1,18 @@
+package stepDefinition;
+
+import cucumber.api.java.en.Then;
+import org.junit.Assert;
+import pages.ProductPage;
+
+public class ProductPageStepDef extends AbstractStepDef {
+
+    ProductPage productPage = new ProductPage(webDriver);
+
+    @Then("^'(.*)' notification is displayed$")
+    public void alertMessageIsDisplayed(String message) {
+        Assert.assertTrue("The alert message is not displayed", productPage.getAlertMessage().isDisplayed());
```

Comment:
What do you need method parameter 'message'?

Reply:
Why doesn't he use this parameter to check that message corresponds to the expected one

Reply:
Because I can't navigate to next page if I will not close this 'alert message'

Reply:
Also, this method will be changed with PR from Andrei

Reply:
1. question was why do you parse a parameter from step as you are not using it
2. closing alert message should be done in another step or you should rename your method/step as they don't match

Reply:
Done